VimeoNetworking/VimeoNetworking/ResponseCache.swift

```diff
@@ -90,19 +99,19 @@ final internal class ResponseCache
             self.cache.setObject(responseDictionary, forKey: key)
         }
         
-        private func responseDictionaryForKey(key: String) -> VimeoClient.ResponseDictionary?
+        private func responseDictionary(forKey key: String, completion: ResponseDictionaryClosure)
```

Comment:
I think the cost of using a closure instead of a return value for the memory cache is greater than the cost of either not using or of modifying the cache protocol.